spec/adapters/jira-spec.js

```diff
@@ -2,237 +2,371 @@ import { JSDOM } from 'jsdom';
+       <div class="sc-guUSXb PaUQl">
+          <div data-react-beautiful-dnd-droppable="2">
+             <div>
+                <div class="sc-eOnLuU XXzdQ" data-react-beautiful-dnd-draggable="2" tabindex="0" data-react-beautiful-dnd-drag-handle="2" aria-roledescription="Draggable item. Press space bar to lift" draggable="false">
+                   <div class="sc-fMvjmP dVeVUp"><img class="sc-gOhbcK jakkJm" role="presentation" src="/secure/viewavatar?size=xsmall&amp;avatarId=10315&amp;avatarType=issuetype"></div>
+                   <div class="sc-igaqVs ioXeXy">
+                      <div>
+                         <div class="sc-bXVTRF coaBE">A Random JIRA Backlog Issue</div>
+                      </div>
+                   </div>
+                   <a class="sc-gcpVEs eofwL" href="/browse/UXPL-39" tabindex="-1">UXPL-39</a>
+                   <div class="sc-eylKsO bqlpkB"></div>
+                </div>
+             </div>
+             <div>
+                <div class="sc-eOnLuU cPUuvS" data-react-beautiful-dnd-draggable="2" tabindex="0" data-react-beautiful-dnd-drag-handle="2" aria-roledescription="Draggable item. Press space bar to lift" draggable="false">
+                   <div class="sc-fMvjmP dVeVUp"><img class="sc-gOhbcK jakkJm" role="presentation" src="/secure/viewavatar?size=xsmall&amp;avatarId=10315&amp;avatarType=issuetype"></div>
+                   <div class="sc-igaqVs ioXeXy">
+                      <div>
+                         <div class="sc-bXVTRF coaBE">Another Random JIRA Issue</div>
+                      </div>
+                   </div>
+                   <a class="sc-gcpVEs eofwL" href="/browse/UXPL-47" tabindex="-1">UXPL-47</a>
+                   <div class="sc-eylKsO bqlpkB"></div>
+                </div>
+             </div>
+          </div>
+          <div class="sc-hBAMER kFCrqp">
+             <button class="sc-iipuKH ckwscm">
+                <span class="sc-bPzAnn fMqaHi">
+                   <span class="sc-csuQGl juTkzQ" aria-label="Create issue">
+                      <svg width="24" height="24" viewBox="0 0 24 24" focusable="false" role="presentation">
+                         <path d="M13 11V7a1 1 0 0 0-2 0v4H7a1 1 0 0 0 0 2h4v4a1 1 0 0 0 2 0v-4h4a1 1 0 0 0 0-2h-4z" fill="currentColor" fill-rule="evenodd"></path>
+                      </svg>
+                   </span>
+                </span>
+                Create issue
+             </button>
+          </div>
+       </div>
+    </div>
+  `;
+
+  const STORYPAGE = `
+  <div>
+    <div>
+      <a href="/browse/UXPL-39"><span>UXPL-39</span></a>
+    </div>
+    <div>
+      <h1>A Random JIRA Board Issue</h1>
+      <div>
+        A referenced issue on the same issue page
+        <a href="/browse/UXPL-47"><span>UXPL-47</span></a>
+      </div>
+    </div>
+  </div>
+  `;
+
+  const BOARD_STORY = `
+  <div role="dialog">
+    <a href="/browse/UXPL-39"><span>UXPL-39</span></a>
+    <h1>A Random JIRA Board Issue</h1>
+  </div>
```

Comment:
Because the HTML is _really_ complex, this is the minimal example that can be read by the implementation. This cuts ~20 levels of divs, and unused interactions.